docker/docker-compose.1604.53.yaml

```diff
@@ -35,6 +35,9 @@ services:
       - MAX_ALLOCS_ALLOWED_read_10000_chunks_from_file=200500 #5.3 improvement 210050
       - MAX_ALLOCS_ALLOWED_udp_1000_reqs_1_conn=16250
       - MAX_ALLOCS_ALLOWED_udp_1_reqs_1000_conn=226050
+      - MAX_ALLOCS_ALLOWED_1000_reqs=6100
+      - MAX_ALLOCS_ALLOWED_1000_tcpbootstraps=4100
```

Comment:
This is a sad regression, doubly so given that it seems to have come in 5.2.

Reply:
I think it's the same one we've discovered 3 times and every time feel sad about it.

Reply:
which is the regression? We know about a 3 allocation regression _per connection_.

Reply:
@weissi - recall we found that there was a regression that was something to do with channel options but which was very hard to look at when we did https://github.com/apple/swift-nio/pull/1496.
Given how hard it is to look at regressions across compiler versions I'm reluctant dive in again.